backend/core/src/seeds/listings/listing-detroit-10145.ts

```diff
@@ -0,0 +1,133 @@
+import { ListingSeedType, PropertySeedType, UnitSeedType } from "./listings"
+import { ListingStatus } from "../../listings/types/listing-status-enum"
+import { CountyCode } from "../../shared/types/county-code"
+import { CSVFormattingType } from "../../csv/types/csv-formatting-type-enum"
+import { ApplicationMethodType } from "../../application-methods/types/application-method-type-enum"
+import { ListingDefaultSeed } from "./listing-default-seed"
+import { UnitCreateDto } from "../../units/dto/unit.dto"
+import { BaseEntity, DeepPartial } from "typeorm"
+import { Listing } from "../../listings/entities/listing.entity"
+import { UnitStatus } from "../../units/types/unit-status-enum"
+import { ApplicationMethod } from "../../application-methods/entities/application-method.entity"
+import assert from "assert"
+
+const mcvProperty: PropertySeedType = {
+  buildingAddress: {
+    city: "Detroit",
+    state: "MI",
+    street: "4701 Chrysler Drive",
+    zipCode: "48201",
+    latitude: 42.35923,
+    longitude: -83.054134,
+  },
+  buildingTotalUnits: 194,
```

Comment:
We haven't been setting buildingTotalUnits (I'm not sure if this could include non-regulated units?); we've relied instead on the `units` table, which it looks like this seed is already populating. Maybe remove this field?

Reply:
I was thinking it might be good to populate it anyway since we have that data. We can decide to use it or not use it in the frontend. Do you see a downside with populating it?

Reply:
Only downside I can think of: if this seed data has it, and that masks some UI issue with the actual data (since the actual data doesn't have it). I don't feel strongly enough about this, though, so find to resolve this comment and move on :)

Note: why doesn't the actual data have this? Because I didn't know whether `property.buildingTotalUnits` was meant to include units across multiple `Listing`s, or units that weren't part of any `Listing` (e.g. if they weren't regulated). Seems less important now that we're moving towards Listing and Property being 1:1

Reply:
Hm, I'm a bit confused about what you mean by "the actual data doesn't have it". Is that not the intended meaning of the Total_Units field in the ArcGIS dataset? As for whether or not the property.buildingTotalUnits was meant to include units across multiple Listings, since we won't ever have multiple Listings per property, won't that mean either way that it's the same result? I'll resolve since you said you don't feel strongly about it.